articles/openshift/concepts-networking.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Concepts - Networking diagram for Azure Red Hat OpenShift 4
+description: Networking diagram and overview for Azure Red Hat OpenShift networking
+author: sakthi-vetrivel
+ms.author: suvetriv
+ms.topic: tutorial
+ms.service: container-service
+ms.date: 06/22/2020
+#Customer intent: As a cluster administrator, I want to understand networking in a Azure Red Hat OpenShift cluster.
+---
+
+# Networking in Azure Red Hat OpenShift 4
+
+This guide covers an overview of networking in Azure Red Hat OpenShift 4 clusters, along with a diagram and a list of important endpoints.
+
+For more details on core OpenShift networking concepts, please see the [Azure Red Hat OpenShift 4 networking documentation](https://docs.openshift.com/aro/4/networking/understanding-networking.html).
+
+## Networking Concepts in Azure Red Hat OpenShift
+
+### Networking Basics in OpenShift
+
+OpenShift Software Defined Networking (SDN) is used to configure an overlay network using Open vSwitch (OVS), an OpenFlow implementation based on Container Network Interface (CNI). The SDN supports different plugins, and the plugin being used in Azure Red Hat OpenShift 4.3 is Network Policy. All network communication is managed by the SDN, so no extra routes are needed on the your Virtual Networks (VNets).
+
+### Azure Red Hat OpenShift Networking Specifics
+
+The following features are specific to Azure Red Hat OpenShift:
+* Bring your own VNet is supported.
+* Pod and Service CIDRs are configurable.
+* Nodes and Masters are in different subnets.
+* Nodes and Masters VNet subnets should be minimum /2.
+* Pod CIDR should be minimum /18 (the CIDR is not routable similar to kubenet).
+* Each node is allocated /23 subnet for its pods (you can not change this value).
+* You can not attach a pod to multiple networks (OpenShift feature).
+* You can not configure Egress static IP (OpenShift feature).
+
+### Network Modes
+
+The following network modes are available in Azure Red Hat OpenShift 4x:
+
+* **API Visibility** - Set the API visibility when running the [az aro create command](tutorial-create-cluster.md#create-the-cluster).
+    * "Public" - API Server is accessible by the public.
+    * "Private" - API Server assigned a private IP from the masters subnet, only accessible using connected networks (Peered VNetS, other subnets in ARO VNet). A private DNS Zone will be created on the customer behalf.
```

Comment:
"ARO" isn't an official acronym. Perhaps change this to "in the cluster's virtual network".
Peered VNetS -> peered virtual networks
on the customer behalf -> on the customer's behalf